index.html

```diff
@@ -23,7 +23,7 @@
         - index.html daj ven iz /src v /root
     -->
 
-
+    <!-- REVIEW -->
     <div class="main">
         <!-- Naslovna glava -->
         <div class="d-flex flex-row">
```

Comment:
Maybe add better naming then "my-auto"
```suggestion
        <div class="d-flex flex-row">
```